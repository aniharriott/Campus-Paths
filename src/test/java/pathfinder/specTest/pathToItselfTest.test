# Test that findMinPath will return 0 cost path

CreateGraph graph1

AddNode graph1 n1
AddNode graph1 n2

AddEdge graph1 n1 n2 6.000

FindPath graph1 n1 n1