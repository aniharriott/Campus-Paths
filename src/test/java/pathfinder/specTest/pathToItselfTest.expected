# Test that findMinPath will return 0 cost path

created graph graph1

added node n1 to graph1
added node n2 to graph1

added edge 6.000 from n1 to n2 in graph1

path from n1 to n1:
total cost: 0.000